### DISCUSSION TITLE
How to add custom cookies to nuclei scans?

### DISCUSSION THREAD

**Discussion by Ma1Dong** [projectdiscovery/nuclei#1623] [Q&A] [answered]
Which parameter should I use if I need to add cookie scanning，thanks

**Comment by No-Github** [comment, marked as answer]
use `-H`

```
nuclei -t test.yaml -u https://target.com -H "Cookie: test"
```

**Comment by Ma1Dong** [comment]
> use `-H`
> 
> ```
> nuclei -t test.yaml -u https://target.com -H "Cookie: test"
> ```

thinks，I am your fan！
